filament/backend/include/backend/platforms/VulkanPlatform.h

```diff
@@ -27,8 +27,18 @@ namespace filament::backend {
 
 class VulkanPlatform : public Platform {
 public:
+    struct SurfaceBundle {
+        void* surface;
+        // On certain platforms, the extent of the surface cannot be queried from Vulkan. In those
+        // situations, we allow the frontend to pass in the extent to use in creating the swap
+        // chains.
```

Comment:
Maybe add a note that `width` and `height` can be set to zero for the default behavior.

Reply:
done